simulator.go

```diff
@@ -285,12 +297,98 @@ func nodeInfoGet(w http.ResponseWriter, request *http.Request) {
 	io.WriteString(w, fixedIP.URLv4())
 }
 
+// getSimulatorID gets the container ID of the simulation container.
+func getSimulatorID(w http.ResponseWriter, request *http.Request) {
```

Comment:
I think adding this is a bit awkward. The only reason we need this is because the simulation container needs to
connect itself to the network. Maybe we could add another API endpoint that connects the simulation to a named network:

- `/testsuite/{suite}/node/{node}/network/{network}` connects the node
- `/testsuite/{suite}/network/connect` connects the simulation

Reply:
We could also remove the extra APIs specific to the simulation and just give special meaning to the `"simulation"` node container ID. Just really trying to avoid the situation where the sim container needs to learn about its own docker container ID. It feels wrong.

So instead of having two APIs for 'connecting nodes' and 'connecting simulation', we could have

`GET /testsuite/{suite}/network/{network}/{node}`
`POST /testsuite/{suite}/network/{network}/{node}`
`DELETE /testsuite/{suite}/network/{network}/{node}`

and if `node` is `"simulation"`, it gets-ip/connects/disconnects the simulation container.